src/tool/xmeta/cpp/xmeta_idl_reader.h

```diff
@@ -22,7 +22,7 @@ namespace xlang::xmeta
         xmeta_idl_reader(std::string_view const& idl_assembly_name)
         {
             m_assembly_names.emplace_back(idl_assembly_name);
-            m_current_assembly = m_assembly_names.back();
+            m_cur_assembly = m_assembly_names.back();
```

Comment:
Let's have either an assembly model, or use the meta reader's Assembly type. Either way, we need to hold more information than just the name.